jax/experimental/jax2tf/examples/tflite/mnist/README.md

```diff
@@ -0,0 +1,92 @@
+# MNIST classification
+
+This directory contains the code necessary to train a simple convolution network on the MNIST dataset, and subsequently convert it using jax2tf into a model that can be converted to tflite format.
+
+This example is based on the [TensorFlow Lite demo of building a handwritten digit classifier app with TensorFlow Lite](https://developer.android.com/codelabs/digit-classifier-tflite). The model code is taken from [FLAX Linen Mnist Example](https://github.com/google/flax/tree/master/linen_examples/mnist).
+
+## Requirements
+
+* TensorFlow dataset mnist will be downloaded and prepared automatically, if necessary
+
+## Training the model
+
+You can train and export the model to tflite format yourself running
+```bash
+$ python mnist.py
+```
+
+The training itself takes only about 1 minute, assuming the dataset has already been downloaded. The dataset will be downloaded directly into a data/ directory in the /tmp/jax2tf/mnist directory. The training loop runs for 3 epochs, and the model is then converted to TensorFLow Lite and subsequently saved into /tmp/jax2tf/mnist/mnist.tflite.
+
+### Convert to TF function using jax2tf converter
+Write a predict function from the trained model and use the jax2tf converter to convert it to a TF function.
+
+```
+def predict(image):
+   return CNN().apply({'params': optimizer.target}, image)
+```
+```
+# Convert Flax model to TF function.
+tf_predict = tf.function(
+    jax2tf.convert(predict, enable_xla=False),
+    input_signature=[
+        tf.TensorSpec(shape=[1, 28, 28, 1], dtype=tf.float32, name='input')
+    ],
+    autograph=False)
+```
+
+### Convert to TensorFlow Lite
+[TensorFlow Lite converter](https://www.tensorflow.org/lite/convert#python_api_) provides a number of ways to convert a TensorFlow model to the TensorFlow Lite format. Here we use the `from_concrete_functions` API.
```

Comment:
Suggestion (having pre-defined TensorFlow Lite as TF Lite before):
- Also, using the second person ("you") (a suggestion from the Google style guide)

```suggestion
### Convert the trained model to the TF Lite format

The [TF Lite converter](https://www.tensorflow.org/lite/convert#python_api_) provides a number of ways to convert a TensorFlow model to the TF format. For example, you can  use the `from_concrete_functions` API as follows:
```